### DISCUSSION TITLE
How to ignore public/something folder when build

### DISCUSSION THREAD

**Discussion by motea927** [nuxt/nuxt#16765] [Questions] [answered]
I put images in public folder to support development, folder is `public/cdn/**.png`,
but I don't want to copy this folder when build,
because I have CDN url in production,
I try to change `Nuxt3` `next.config.ts.ignore` equal to `/public/cdn`,
it seems not support.

**Comment by sevillaarvin** [comment]
Have you tried the `cdnURL` option? https://v3.nuxtjs.org/api/configuration/nuxt.config#cdnurl

**Reply by motea927** [reply]
Hi, When local dev, I don't want to use Cdn url, I want to use local public/Cdn 's file

**Comment by bot08** [comment]
You can use a .nuxtignore file to let Nuxt ignore layout, pages, components, composables and middleware files in your project’s root directory (rootDir) during the build phase. Looks like Public directory isn’t supposed.

**Reply by motea927** [reply]
Yes, seems like Public directory isn’t supposed.

**Comment by motea927** [comment, marked as answer]
This way has worked for me,
change `nuxt.config.ts` 's public,
And use different public folder for dev and production.

```typescript
dir: {
    public: process.env.NODE_ENV === 'development' ? 'public_dev' : 'public',
},
```
